Description: Remove unused build dep and features
 Removed features are empty and have no corresponding code.
Last-Update: 2022-09-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,14 +22,7 @@
 repository = "https://github.com/purpleprotocol/mimalloc_rust"
 [dependencies.libmimalloc-sys]
 version = "0.1.25"
-default-features = false
 
 [features]
-debug = ["libmimalloc-sys/debug"]
-debug_in_debug = ["libmimalloc-sys/debug_in_debug"]
-default = ["secure"]
-local_dynamic_tls = ["libmimalloc-sys/local_dynamic_tls"]
-override = ["libmimalloc-sys/override"]
-secure = ["libmimalloc-sys/secure"]
 [badges.travis-ci]
 repository = "purpleprotocol/mimalloc_rust"
